kedro-telemetry/kedro_telemetry/plugin.py

```diff
@@ -90,7 +122,7 @@ def before_command_run(
             main_command = masked_command_args[0] if masked_command_args else "kedro"
 
             logger.debug("You have opted into product usage analytics.")
-            hashed_username = _get_hashed_username()
+            hashed_username = _get_or_create_uuid()
```

Comment:
Should the variable be renamed to `user_uuid` or something like that now it's not a hashed username anymore?